x-pack/plugins/lens/public/datatable_visualization/components/table_basic.scss

```diff
@@ -2,7 +2,8 @@
   height: 100%;
 }
 
-.lnsTableCell {
+// only apply truncation inside of a table cell (not within popover)
+[data-test-subj='dataGridRowCell'] .lnsTableCell {
```

Comment:
I feel like we shouldn't be using the `data-test-subj` attribute as a selector for styles. Would it be possible to apply a new class instead?